examples/rotator_test_light.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import logging
+import yaml
+
+import numpy as np
+from matplotlib import pyplot as plt
+import astropy.units as u
+from astropy.time import Time
+from astropy.coordinates import AltAz, ICRS, EarthLocation, Angle, FK5
+import asyncio
+from lsst.ts import salobj
+from lsst.ts.idl.enums import ATPtg
```

Comment:
why ATPtg ? This is script for the MT rotator... 
You use this multiple places below...

Reply:
The MTPtg does not have the idl enums defined yet. Or at least, it didn't when I wrote this script, so I stole from the ATPtg... I wish we would have only one pointing component, which means we would not have to duplicate this... :(

Reply:
Ok that's reasonable. So just add a comment in the code saying why MTPtg isn't used and that'll be sufficient
.

Reply:
done